### ISSUE TITLE
Unexpected token { because ES10 syntax in fs-extra

### ISSUE THREAD

**Issue by ayame30** [seppevs/migrate-mongo#202]
I am currently using node 8.11.3, and I am facing unexpected token { issue when I use migrate-mongo init

How to reproduce
node 8.11.3 and migrate-mongo init

```
node@b531b9ebb1eb:/opt/reaction/src$ npm run migrate

> reaction@2.7.0 migrate /opt/reaction/src
> migrate-mongo init

/opt/reaction/src/node_modules/migrate-mongo/node_modules/fs-extra/lib/mkdirs/make-dir.js:86
      } catch {
              ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/opt/reaction/src/node_modules/migrate-mongo/node_modules/fs-extra/lib/mkdirs/index.js:3:44)
```

fs-extra seems using optional catch binding which only supported in ES10.

**Comment by AnthonyTsang** [comment]
same issue

**Comment by seppevs** [comment]
Fixed in v7.2.1
I've downgraded the fs-extra dependency. 

Please try to upgrade to Node 10 or higher. [Node 8 is not maintained anymore](https://nodejs.org/en/about/releases/).

**Comment by ayame30** [comment]
Thank you for the fix
